src/mad_spark_alt/evolution/operators.py

```
@@ -155,7 +163,7 @@ def validate_config(self, config: Dict[str, Any]) -> bool:
         return True  # No specific config needed
 
     async def mutate(
-        self, idea: GeneratedIdea, mutation_rate: float, context: Optional[str] = None
+        self, idea: GeneratedIdea, mutation_rate: float, context: Optional[str] = None,
```

Ruff (0.12.2)

Unused method argument: `context`

(ARG002)